### PR TITLE
Create Pre-Release Issue Template

### PR DESCRIPTION
**Pull request by JackEAllen**
### Issue

Create Pre-Release Checklist Issue Template

### Description

pre-release issue template to ensure we don't miss any pre-release steps and to standardise the approach taken in the build-up to a release of Mantid Imaging

### Acceptance Criteria 

 - [ ] Whole team agrees on format
 - [ ] Grammatically correct
 - [ ] Structure makes sense (order of steps)
 - [ ] Template details all required steps for a re-release to some level.

### Documentation

NA

### CODE REVIEW

File: .github/ISSUE_TEMPLATE/release.md [hunk 1]
```diff
@@ -0,0 +1,81 @@
...
+
+## Description
+
+As part of preparations for the release of Mantid Imaging version [**VERSION***], the following tasks need to be completed.
+
+
+## High Priority Bugs
+<!-- Make sure any high priority bugs which are found during smoke testing are resolved -->
+- [ ] [High priority bugs](https://github.com/mantidproject/mantidimaging/issues?)
+
```
Comment (samtygier-stfc) [inline .github/ISSUE_TEMPLATE/release.md:19]:
```
## Create release branch
- [ ] Create branch from main named release-x.x.x
- [ ] Check that branch protection rules are being applied
```

File: .github/ISSUE_TEMPLATE/release.md [hunk 2]
```diff
@@ -0,0 +1,81 @@
+---
+
+name: Pre-Release Issue Template
+about: Should be used when Preparing for a release to make sure no tasks are missed during the build-up for a release
+title: 'Mantid Imaging [V.V.V] Pre-Release'
+labels: 'Release 🚀'
+assignees: ''
+
```
Comment (samtygier-stfc) [inline .github/ISSUE_TEMPLATE/release.md:8]: If this shows up as a template when creating a new issue, is there a way to have it only visible to devs?

Comment (samtygier-stfc) [reply .github/ISSUE_TEMPLATE/release.md:8]: > I would be in favour of just leaving public
That's fine

Comment (JackEAllen) [reply .github/ISSUE_TEMPLATE/release.md:8]: I've updated the title to hopefully make the template obvious that it is for developers only

**Comment by JackEAllen** [comment]
See in rich display format to view template as it would look negating metadata table at the top if implemented as a issue template: https://github.com/mantidproject/mantidimaging/pull/2059/files?short_path=4482ca0#diff-4482ca0941ca8f0e3db46d20770ecf9158cc00322eee6e760460c47a4a5932c2

**Review by samtygier-stfc — APPROVED** [review]
Happy with this. We can update if we find anything missing.

**Pull request merged**